src/Infrastructure.EntityFramework/AdminConsole/Repositories/Queries/OrganizationUserUserViewQuery.cs

```diff
@@ -28,9 +28,9 @@ from su in su_g.DefaultIfEmpty()
             SsoExternalId = x.su.ExternalId,
             Permissions = x.ou.Permissions,
             ResetPasswordKey = x.ou.ResetPasswordKey,
-            UsesKeyConnector = x.u != null && x.u.UsesKeyConnector,
+            UsesKeyConnector = x.u.UsesKeyConnector,
             AccessSecretsManager = x.ou.AccessSecretsManager,
-            HasMasterPassword = x.u != null && !string.IsNullOrWhiteSpace(x.u.MasterPassword)
+            HasMasterPassword = !string.IsNullOrWhiteSpace(x.u.MasterPassword)
```

Comment:
Generated SQLite from before change

```sql
SELECT "o"."Id",
  "o"."UserId",
  "o"."OrganizationId",
  "u"."Name",
  COALESCE("u"."Email", "o"."Email") AS "Email",
  "u"."AvatarColor",
  "u"."TwoFactorProviders",
  "u"."Premium",
  "o"."Status",
  "o"."Type",
  "o"."ExternalId",
  "s"."ExternalId" AS "SsoExternalId",
  "o"."Permissions",
  "o"."ResetPasswordKey",
  "u"."Id" IS NOT NULL AND "u"."UsesKeyConnector" AS "UsesKeyConnector",
  "o"."AccessSecretsManager",
  "u"."Id" IS NOT NULL AND "u"."MasterPassword" IS NOT NULL AND trim("u"."MasterPassword") <> 'P|CfDJ8KUSA3v4Y1dFg8Rnf6FjA5fdvqaglKtClSQ1I3diP2vxzTdwq2_dCgDHF-chiyxdEEkJZ_XZnvZFhIpQGoqQL4f35sOzasRWeL8HXy0O59nSBVpX2WOp565OeY4fYRwuNg' AS "HasMasterPassword"
FROM "OrganizationUser" AS "o"
LEFT JOIN "User" AS "u" ON "o"."UserId" = "u"."Id"
LEFT JOIN "SsoUser" AS "s" ON "o"."UserId" = "s"."UserId" AND "o"."OrganizationId" = "s"."OrganizationId"
```

After change:
```sql
SELECT "o"."Id",
  "o"."UserId",
  "o"."OrganizationId",
  "u"."Name",
  COALESCE("u"."Email",
  "o"."Email") AS "Email",
  "u"."AvatarColor",
  "u"."TwoFactorProviders",
  "u"."Premium",
  "o"."Status",
  "o"."Type",
  "o"."ExternalId",
  "s"."ExternalId" AS "SsoExternalId",
  "o"."Permissions",
  "o"."ResetPasswordKey",
  "u"."UsesKeyConnector",
  "o"."AccessSecretsManager",
  "u"."MasterPassword" IS NOT NULL AND trim("u"."MasterPassword") <> 'P|CfDJ8KUSA3v4Y1dFg8Rnf6FjA5dJYW3Rjd8aosV-K01yVWmZVkVbItCW4oOuqXMvKO2PD9DIw0XKjy8sqgWtIKF1q7cT1JKgc3F2mwImAtRqP5WfrlURovVo9WdKmv2sI9lHUg' AS "HasMasterPassword"
FROM "OrganizationUser" AS "o"
LEFT JOIN "User" AS "u" ON "o"."UserId" = "u"."Id"
LEFT JOIN "SsoUser" AS "s" ON "o"."UserId" = "s"."UserId" AND "o"."OrganizationId" = "s"."OrganizationId"
```

And the equivalent hand wrote SQL:
```sql
SELECT
    OU.[Id],
    OU.[UserId],
    OU.[OrganizationId],
    U.[Name],
    ISNULL(U.[Email], OU.[Email]) Email,
    U.[AvatarColor],
    U.[TwoFactorProviders],
    U.[Premium],
    OU.[Status],
    OU.[Type],
    OU.[AccessSecretsManager],
    OU.[ExternalId],
    SU.[ExternalId] SsoExternalId,
    OU.[Permissions],
    OU.[ResetPasswordKey],
    U.[UsesKeyConnector],
    CASE WHEN U.[MasterPassword] IS NOT NULL THEN 1 ELSE 0 END AS HasMasterPassword
FROM
    [dbo].[OrganizationUser] OU
LEFT JOIN
    [dbo].[User] U ON U.[Id] = OU.[UserId]
LEFT JOIN
    [dbo].[SsoUser] SU ON SU.[UserId] = OU.[UserId] AND SU.[OrganizationId] = OU.[OrganizationId]
```

So this seems like a good change to me.

Reply:
I also double checked this because I was concerned about orgUsers in the invited state that are not linked to a user, but they're excluded by the inner join on `User.Id`. So we should always have a User available. And as has been pointed out, we access `x.u` before this anyway, so clearly we do.